### ISSUE TITLE
Proxmox LXC's centos 7 cron job deployment issue

### ISSUE THREAD

**Issue by ricky0221** [ansible-tw/AMA#6]
### Environment：
CentOS 7.2
ansible 2.1.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides

Use the task as below  
```
 - name: ntp update cronjob
          cron: name="NTP update" minute="*/15"  user="root" job="/sbin/ntpdate -s time.stdtime.gov.tw"
```

will encounter this error 

```
TASK [common : ntp update cronjob] *********************************************
fatal: [10.1.1.138]: FAILED! => {"changed": false, "failed": true, "module_stderr": "", "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_f7D6qv/ansible_module_cron.py\", line 700, in <module>\r\n    main()\r\n  File \"/tmp/ansible_f7D6qv/ansible_module_cron.py\", line 575, in main\r\n    crontab = CronTab(module, user, cron_file)\r\n  File \"/tmp/ansible_f7D6qv/ansible_module_cron.py\", line 233, in __init__\r\n    self.read()\r\n  File \"/tmp/ansible_f7D6qv/ansible_module_cron.py\", line 251, in read\r\n    (rc, out, err) = self.module.run_command(self._read_user_execute(), use_unsafe_shell=True)\r\n  File \"/tmp/ansible_f7D6qv/ansible_module_cron.py\", line 480, in _read_user_execute\r\n    elif os.getlogin() != self.user:\r\nOSError: [Errno 25] Inappropriate ioctl for device\r\n", "msg": "MODULE FAILURE"}

```

**Comment by ricky0221** [comment]
When update Ansible to 2.2.0.0, this issue would be resolved.

ansible 2.2.0.0
  config file = /home/aplus/lamp-ansible/ansible.cfg
  configured module search path = Default w/o overrides

**Comment by chusiang** [comment]
This issue has be fixed with upgrade ansible to v2.2.0.0.

Good job!
